Documentation/Contributing/UnitTests.md

```diff
@@ -1,21 +1,19 @@
 
-# MRTK Tests
-
-MRTK has a set of tests to ensure that changes to our code do not regress existing behavior. When you add a new feature, please:
+# Writing and Running Tests in MRTK
+MRTK has a set of tests to ensure that changes to our code do not regress existing behavior. Before you submit a pull request, make sure to :
 
 1. Run the tests locally to make sure your changes don't regress existing behavior (you will not be able to check in if any tests fail)
-2. Write new tests to ensure that other people don't break your feature in the future.
 
-If you fix a bug, please consider writing a test to ensure that this bug doesn't regress in the future as well.
+2. If fixing a bug, write a test to prevent others from breaking your bugfix.
 
-## Executing tests locally
+2. If writing a feature, write new tests to ensure that other people don't break your feature in the future.
 
-The [Unity Test Runner](https://docs.unity3d.com/Manual/testing-editortestsrunner.html) can be found under
- Window > General > Test Runner and will show all available MRTK play and edit mode tests.
+## Running tests
+### Running tests from Unity editor
+The [Unity Test Runner](https://docs.unity3d.com/Manual/testing-editortestsrunner.html) can be found under Window > General > Test Runner and will show all available MRTK play and edit mode tests. 
 
-You can also run the [powershell](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell?view=powershell-6)
-script located at `Scripts\test\run_playmode_tests.ps1`. This will run the playmode tests exactly as they are executed
-on github / CI (see below), and print results. Here are some examples of how to run the script
+### Running tests from command line
+You can also run the [powershell](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell?view=powershell-6) script located at `Scripts\test\run_playmode_tests.ps1`. This will run the playmode tests exactly as they are executed on github / CI (see below), and print results. Here are some examples of how to run the script
```

Comment:
Oh, I didn't know about that! I never actually tested that experience. I suppose you could just run that command line tests multiple times and parse output. I think the approach of modifying the code is still faster since every time you run a test from cmd unity does a bunch of extra work. I can mention the command line arguments though.

Reply:
Exactly they have different benefits. The python script is great to target one test case and get multiple results in editor but you have to run and copy and paste and then delete/cleanup

Where a command line of run everything 5-10 times while I go to lunch involves less setup and covers more bases for that need

Reply:
Done, see run_repeat_tests.ps1